pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/WorkflowDao.sql.stg

```diff
@@ -4,12 +4,16 @@ queryOrderedActiveNextStateCandidatesByStudyIdAndFromStateId() ::= <<
 select trans.workflow_transition_id,
        next_state_type.workflow_state_type_code,
        next_act_state.study_activity_id,
+       next_redirect_state.study_guid,
+       next_redirect_state.redirect_url,
        expr.expression_text
   from workflow_transition as trans
   join workflow_state as next_state on next_state.workflow_state_id = trans.next_state_id
   join workflow_state_type as next_state_type on next_state_type.workflow_state_type_id = next_state.workflow_state_type_id
   left join workflow_activity_state as next_act_state on next_act_state.workflow_state_id = next_state.workflow_state_id
   left join expression as expr on expr.expression_id = trans.precondition_expression_id
+  left join workflow_study_redirect_state as next_redirect_state on next_redirect_state.workflow_state_id = next_state.workflow_state_id
+  join umbrella_study as us on us.umbrella_study_id = trans.umbrella_study_id
```

Comment:
Don't need this `join` anymore.

Reply:
We do need the join to workflow_study_redirect_state to get studyGuid and redirectUrl.

Reply:
D'oh. Oh right, I missed that.

Reply:
Wait no. I meant we don't need the `join` to umbrella_study. We can keep the `left join`.

Reply:
Yeah removed that join